src/coreclr/vm/ilstubresolver.cpp

```diff
@@ -558,4 +558,44 @@ void ILStubResolver::StubGenFailed(ILStubResolver* pResolver)
     }
 
     pResolver->ClearCompileTimeState(ILNotYetGenerated);
-}
\ No newline at end of file
+}
+
+#ifndef DACCESS_COMPILE
+void FinalizeILStub(
```

Comment:
Should this be an instance method on ILStubResolver?

Reply:
It can be. The only issue is the calling order. Standalone functions tend to avoid that sort of thing and since this function doesn't need internal state it not being a member made sense. I'm find changing it to a member if you'd prefer though.

Reply:
I think it would look better as a member.

Reply:
Done.